### DISCUSSION TITLE
How should I add new blogs ?

### DISCUSSION THREAD

**Discussion by siyehua** [thuliteio/doks#1203] [Q&A] [answered]
<img width="488" alt="image" src="https://github.com/gethyas/doks/assets/9376247/b53b18ad-dee2-4f91-8591-190feea675fd">

I add the markdown files like this, but when I click the Blog page, it only has two blogs:

<img width="458" alt="image" src="https://github.com/gethyas/doks/assets/9376247/6a8cdd73-a5ca-4931-9996-72b4f80c0404">

<img width="1345" alt="image" src="https://github.com/gethyas/doks/assets/9376247/e02d544c-3772-4e3b-83ee-8785b342585e">

I want to show the `create` and `use` blogs

**Comment by h-enk** [comment, marked as answer]
Move the `create` and `use` folders one level up (drop the `aigc` folder) — so they are on the same level as the `example` folder. See also [Page bundles](https://gohugo.io/content-management/organization/#page-bundles)

**Comment by siyehua** [comment]
Thanks, I know it
